### DISCUSSION TITLE
参考文献序号消失，变成了标签

### DISCUSSION THREAD

**Discussion by MichaelJJ008** [sjtug/SJTUThesis#809] [提问]
![8bd427519a0620923b4604a1fc132d0](https://user-images.githubusercontent.com/71876253/169651011-7c85a30c-f786-4cb3-93a5-36a6c8683328.png)

**Comment by skyzh** [comment]
请参见“常见问题”的“为什么参考文献找不到 citation？”

https://github.com/sjtug/SJTUThesis/wiki/常见问题

**Reply by MichaelJJ008** [reply]
我是Windows系统，使用的texlive2022+texstudio，适用吗

**Reply by MichaelJJ008** [reply]
我用biber --cache找到了并删除了缓存文件夹，重新编译好像没用

**Reply by skyzh** [reply]
怎么编译的？

**Reply by MichaelJJ008** [reply]
texstudio里编译并生成pdf的

**Reply by MichaelJJ008** [reply]
日志里是Please (re)run Biber on the file:(biblatex) main(biblatex) and rerun LaTeX afterwards.

**Comment by AlexaraWu** [comment]
参见 [模板使用 -- TexStudio 用户](https://github.com/sjtug/SJTUThesis#texstudio-%E7%94%A8%E6%88%B7)。

**Reply by MichaelJJ008** [reply]
这两项设置一直是和链接里教程一致的

**Comment by MichaelJJ008** [comment]
我是在bib文件中将自己引用文献的信息粘贴进去，也保留了模板里自带的20篇文献信息。
刚才我试了一下将正文中我引用自己文献的内容部分注释掉，只保留模板里作为示例的一部分，重新编译就能正常显示序号了。
这之后我取消注释，我自己的文献序号也正常了。
所以现在暂时好了，不过不知道下一次什么时候出现这种问题

**Comment by adaxidedakaonang** [comment]
我在撰写西安电子科技大学博士论文时遇到了同样的问题，表现为参考文献的引用显示的是标签名称而不是序号。可能的解决方案也很简单，就是重复编译多次（四五次即可），基本就能显示成序号了。
